pkg/config/config.go

```
@@ -391,7 +391,7 @@ func processConfigFile(
 	defer func(reader *os.File) {
 		err := reader.Close()
 		if err != nil {
-			u.LogWarning(atmosConfig, fmt.Sprintf("error closing file '"+configPath+"'. "+err.Error()))
+			u.LogWarning(fmt.Sprintf("error closing file '" + configPath + "'. " + err.Error()))
```

golangci-lint (1.62.2)

printf: non-constant format string in call to fmt.Sprintf

(govet)